basics/indexing/native-text-index.md

```diff
@@ -1,38 +1,32 @@
+## How is Pinot different?
 
-## How Is Pinot Different?
+Pinot, like any other database/OLAP engine, does not need to conform to the entire full text search domain-specific language (DSL) that is traditionally used by full-text search (FTS) engines like ElasticSearch and Solr. In traditional SQL text search use cases, the majority of text searches belong to one of three patterns: prefix wildcard queries (like `pino*`), postfix or suffix wildcard queries (like `*inot`), and term queries (like `pinot`).
 
-Pinot, or any other database/OLAP engine, do not need to conform to the entire full text search DSL that is traditionally used by FTS engines like ElasticSearch and Solr. Looking at traditional SQL like text search use cases, majority of text searches comprise of three patterns -- prefix wildcard queries, postfix wildcard queries and term queries.
+## Native text indices in Pinot
 
-## Native Text Indices in Pinot
+In Pinot, native text indices are built from the ground up. They use a custom text-indexing engine, coupled with Pinot's powerful inverted indices, to provide a fast text search experience.
 
-Native text indices are built from the ground up. They use a custom text indexing engine, coupled with Pinot's powerful inverted indices, to provide a super fast text search experience.
+The benefits are that native text indices are 80-120% faster than Lucene-based indices for the text search use cases mentioned above. They are also 40% smaller on disk.
 
-## Benefits of Native Text Indices
+Native text indices support real-time text search. For `REALTIME` tables, native text indices allow data to be indexed in memory in the text index, while concurrently supporting text searches on the same index.
 
-Native text indices are 80-120% faster than Lucene based indices for the text search use cases mentioned above. They are also 40% smaller on disk.
+Historically, most text indices depend on the in-memory text index being written to first and then sealed, before searches are possible. This limits the freshness of the search, being near-real-time at best.
 
-## Real Time Indexing And Search
-
-A new feature that native text indices support are real time text search. For REALTIME tables, native text indices allow data to be indexed in memory in the text index, while concurrently supporting text searches on the same index.
-
-Historically, most text indices depend on the in memory text index being written to first and then sealed, before searches are possible. This limits the freshness of the search, being near real time at best.
-
-Native text indices come with a custom in memory text index, which allows for real time indexing and search.
+Native text indices come with a custom in-memory text index, which allows for real-time indexing and search.
 
 ## Searching Native Text Indices
```

Comment:
are we using the term indices intentionally here, or do we want to say indexes?

Reply:
I thought it was intentional, but I'll see if I can find out.